packages/blocks/src/api/registration.js

```diff
@@ -158,7 +158,7 @@ export function registerBlockType( name, settings ) {
 		);
 		return;
 	}
-	if ( select( 'core/blocks' ).getBlockType( name ) ) {
+	if ( ! module.hot && select( 'core/blocks' ).getBlockType( name ) ) {
```

Comment:
Hmm, my gut reaction is that we're "tainting" key packages because of the playground's HMR functionality. Anyway we can curb this? Even though Webpack should be able to handle this reference when producing different builds, I wonder if the control is in the right hands. Putting it differently: an implementation detail of a developer tool is leaking into the main source code.

Reply:
Agreed, but unfortunately this is how the HMR API works.

We could write a Babel plugin that inserts it, haha, but that seems like over-engineering.